tools/docs/manager.ts

```diff
@@ -14,8 +18,8 @@ const noCategoryID = 'no-category';
 const noCategoryDisplayName = 'No Category';
 
 function getTitle(manager: string, displayName: string): string {
-  if (manager === 'regex') {
-    return `Custom Manager Support using Regex`;
+  if (isCustomManager(manager)) {
+    return `Custom Manager Support using ${manager}`;
```

Comment:
this now causes lowercase `regex` 🤔

I think we need to pass a proper display name instead